op-e2e/config/init.go

```diff
@@ -370,14 +374,14 @@ func defaultIntent(root string, loc *artifacts.Locator, deployer common.Address,
 			"faultGameAbsolutePrestate":                defaultPrestate.Hex(),
 			"faultGameMaxDepth":                        50,
 			"faultGameClockExtension":                  1,
-			"faultGameMaxClockDuration":                1200,
+			"faultGameMaxClockDuration":                60,
```

Comment:
we still want the default to use sane parameters. The fast alloc type should override these.
```suggestion
			"faultGameMaxClockDuration":                1200,
```

Reply:
fixed, had to tweak the fast game params for it to work